maths/maths.py

```diff
@@ -0,0 +1,138 @@
+
+# package maths
+# Implement some 
+# trigonometric algorithm
+
+Pi = 3.14159265358979323846
+
+
+def _abs(value):
+    # >>> _abs(-10)
+    # 10
+    # >>> _abs(10)
+    # 10
+    if value < 0:
+        value = -value
+    else:
+        value = value
+
+    return value
```

Comment:
return -value if value < 0 else value

Reply:
I want to push this part of the code to this algorithm warehouse, what should I do